sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableServiceAsyncClient.java

```diff
@@ -58,8 +58,7 @@
 
 import static com.azure.core.util.FluxUtil.monoError;
 import static com.azure.core.util.FluxUtil.withContext;
-import static com.azure.data.tables.implementation.TableUtils.applyOptionalTimeout;
-import static com.azure.data.tables.implementation.TableUtils.swallowExceptionForStatusCode;
+import static com.azure.data.tables.implementation.TableUtils.*;
```

Comment:
We don't use wildcard imports in the SDK. You might want to update your editor's preferences to avoid this, IntelliJ has pretty aggressive defaults about it.

Reply:
Ok, do you happen to know where that setting is so I can disable it? @vcolin7

Reply:
Nevermind I found it